internal/db/migrations/postgres/0/100_server_tags_migrations.up.sql

```diff
@@ -0,0 +1,101 @@
+begin;
+
+-- This series of expressions fixes the primary key on the server table
+alter table session
+  drop constraint session_server_id_server_type_fkey;
+alter table server
+  drop constraint server_pkey;
+alter table server
+  drop column name;
+alter table server
+  add primary key (private_id);
+alter table server
+  add constraint server_id_must_not_be_empty
+  check(length(trim(private_id)) > 0);
```

Comment:
This will not check any data currently in the table. You might want to add a query to verify the existing data complies to the constraint.

Reply:
It will, because we've only ever set name and private_id to the same value, and name had this constraint.